### ISSUE TITLE
Allow <symbol> as a child of <rend>

### ISSUE THREAD

**Issue by craigsapp** [rism-digital/verovio#3063]
Currently `<symbol>` are ignored when contained in `<rend>` (MEI does allow this but verovio ignores them).   It would be useful to allow this, at least when the symbol is smufl-based.

Here is an example:

<img width="464" alt="Screen Shot 2022-10-06 at 1 45 02 PM" src="https://user-images.githubusercontent.com/3487289/194415180-12d836cb-c48e-47d9-97de-18de060d4098.png">

The second tempo marking should be quarter note = 63, which is inside of an rend with 50% fontsize.

The first tempo marking has no rend:

```xml
   <tempo xml:id="tempo-L1F1" staff="1" tstamp="1.000000">
        <symbol glyph.name="metNoteHalfUp" glyph.auth="smufl"/>
        = 120
    </tempo>
```

And the second one is contained in a rend to make the tempo marking smaller:

```xml
       <tempo xml:id="tempo-L5F1" staff="1" tstamp="1.000000">
        <rend xml:id="rend-L5F1" fontsize="50%">
           <symbol glyph.name="metQuarterHalfUp" glyph.auth="smufl"/>
           = 63
        </rend>
      </tempo>
```

The font metrics of the quarter note are being used somehow on the second tempo marking (but perhaps just the total height and not considering the baseline).

<details>
<summary> Click to view MEI data for above example. </summary>

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/dev/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/dev/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="5.0.0-dev">
 <meiHead>
  <fileDesc>
   <titleStmt>
    <title />
   </titleStmt>
   <pubStmt />
  </fileDesc>
  <encodingDesc>
   <appInfo>
    <application isodate="2022-10-06T12:49:22" version="3.13.0-dev-9c1b56b">
     <name>Verovio</name>
     <p>Transcoded from Humdrum</p>
    </application>
   </appInfo>
  </encodingDesc>
  <workList>
   <work>
    <title />
   </work>
  </workList>
  <extMeta>
   <frames xmlns="http://www.humdrum.org/ns/humxml">
    <metaFrame n="0" token="!!!OMD: [half] = 120" xml:id="L1">
     <frameInfo>
      <startTime float="0" />
      <frameType>reference</frameType>
      <referenceKey>OMD</referenceKey>
      <referenceValue>[half] = 120</referenceValue>
     </frameInfo>
    </metaFrame>
   </frames>
  </extMeta>
 </meiHead>
 <music>
  <body>
   <mdiv xml:id="m1jqc0e0">
    <score xml:id="sosaya2">
     <scoreDef xml:id="s1mmre9b" midi.bpm="400.000000">
      <staffGrp xml:id="s1m6ppxk">
       <staffDef xml:id="staffdef-L2F1" n="1" lines="5">
        <clef xml:id="cps382x" shape="G" line="2" />
        <meterSig xml:id="metersig-L3F1" count="2" unit="2" />
       </staffDef>
      </staffGrp>
     </scoreDef>
     <section xml:id="section-L2F1">
      <measure xml:id="measure-L1">
       <staff xml:id="staff-L2F1" n="1">
        <layer xml:id="layer-L1F1N1" n="1">
         <note xml:id="note-L5F1" dur="1" oct="4" pname="c" accid.ges="n" />
        </layer>
       </staff>
       <tempo xml:id="tempo-L1F1" staff="1" tstamp="1.000000">
        <symbol glyph.name="metNoteHalfUp" glyph.auth="smufl"/>
        = 120
       </tempo>
      </measure>
      <scoreDef xml:id="s1oukccu">
       <staffGrp xml:id="sdu9fm7">
        <staffDef xml:id="s7krtpj" n="1">
         <meterSig xml:id="m1gzth02" count="4" unit="4" />
        </staffDef>
       </staffGrp>
      </scoreDef>
      <measure xml:id="measure-L6">
       <staff xml:id="staff-L6F1N1" n="1">
        <layer xml:id="layer-L6F1N1" n="1">
         <note xml:id="note-L9F1" dur="1" oct="4" pname="d" accid.ges="n" />
        </layer>
       </staff>
       <tempo xml:id="tempo-L5F1" staff="1" tstamp="1.000000">
        <rend xml:id="rend-L5F1" fontsize="50%">
         <symbol glyph.name="metNoteQuarterUp" glyph.auth="smufl"/>
         = 63
        </rend>
       </tempo>
      </measure>
     </section>
    </score>
   </mdiv>
  </body>
 </music>
</mei>
```

</details>

**Comment by lpugin** [comment]
I think [this PR](https://github.com/rism-digital/verovio/pull/3058) solves your issue. Could you review it?

**Comment by lpugin** [comment]
Solved with #3058
